rest-api-spec/src/main/resources/rest-api-spec/api/open_point_in_time.json

```diff
@@ -55,6 +56,9 @@
         "description": "Specific the time to live for the point in time",
         "required": true
       }
+    },
+    "body":{
+      "description":"An index filter specified with the Query DSL"
```

Comment:
I wen't for specifying `index_filter`. I was a bit on the fence, hence I left a vague description, because this is not supposed to be full documentation for the request body, it only tells whether a body is supported or not. Say that we add more fields to the body later, like it happened with field_caps, we most likely won't come back to this and include their description in the API spec.